src/xs_driver.cpp

```diff
@@ -414,6 +414,13 @@ bool InterbotixDriverXS::write_commands(
           get_group_info(name)->joint_names.at(i),
           commands.at(i));
       }
+
+      // Apply mechanical reduction to the angular values
+      float j_reduction = js_mech_reduction_map[get_group_info(name)->joint_ids.at(i)];
+      for (size_t i{0}; i < commands.size(); i++) {
+        commands.at(i) = commands.at(i) / j_reduction;
```

Comment:
Gear ratio is now 0.25 (output/input). So the cmd is divided by ratio, so that the angular value increases by 4x.